README.rivet.tpl.md

```diff
@@ -32,16 +32,16 @@
 
 Public-facing projects:
 
-- **Rivet Engine** (you are here): Engine that powers Rivet Actors at scale
+- **Rivet Engine** (you are here): Engine that powers RivetKit at scale
 - **[RivetKit](https://github.com/rivet-gg/rivetkit)**: Lightweight TypeScript library for building Rivet Actors — works with Redis or Rivet Engine
-- **[Rivet Studio](/frontend/apps/studio)**: Like Postman, but for Rivet Actors
+- **[Rivet Inspector](/frontend/apps/studio)**: Like Postman, but for Rivet Actors
 - **[Rivet Hub](/frontend/apps/hub)**: UI for Rivet Engine
 - **[Rivet Documentation](/site/src/content/docs)**
 
 Projects powering Rivet Engine:
 
-- **[Pegboard](packages/edge/services/pegboard/)**: Actor orchestrator
-- **[Guard](packages/edge/infra/guard/)**: Proxy for routing traffic to Rivet Actors
+- **[Pegboard](packages/services/pegboard/)**: Actor orchestrator
+- **[Guard](packages/core/guard/)**: Proxy for routing traffic to Rivet Actors
 - **[Chirp](packages/common/chirp-workflow/)**: Core workflow engine that powers Rivet
```

Comment:
see changes that you overwrote from max on readme

he forgot to update the tpl